controllers/managedocs_controller_test.go

```diff
@@ -363,6 +369,29 @@ var _ = Describe("ManagedOCS controller", func() {
 				Expect(k8sClient.Delete(ctx, secret)).Should(Succeed())
 			})
 		})
+		When("there is a rook-ceph-operator-config ConfigMap", func() {
+			It("should ensure there are RBD CSI resource limits", func() {
+				configMap := rookConfigMapTemplate.DeepCopy()
+				Expect(k8sClient.Create(ctx, configMap)).Should(Succeed())
+
+				Eventually(func() bool {
+					err := k8sClient.Get(ctx, utils.GetResourceKey(configMap), configMap)
+					if err != nil {
+						return false
+					}
+
+					if _, ok := configMap.Data["CSI_RBD_PROVISIONER_RESOURCE"]; !ok {
+						return false
+					}
+
+					if _, ok := configMap.Data["CSI_RBD_PLUGIN_RESOURCE"]; !ok {
+						return false
+					}
+
+					return true
```

Comment:
1. I think we should also check that the value of the key is not an empty string.
2. you are not testing the keys for cephfs

```suggestion
		return configMap.Data["CSI_RBD_PROVISIONER_RESOURCE"] != "" &&
		     configMap.Data["CSI_RBD_PLUGIN_RESOURCE"] != "" &&
		     configMap.Data["CSI_CEPHFS_PROVISIONER_RESOURCE"] != "" &&
		     configMap.Data["CSI_CEPHFS_PLUGIN_RESOURCE"] != ""
```